### PR TITLE
Feat/stock view

### PR DESCRIPTION
**Pull request by sengfer-ISST**
Initiating Stock View
- Creating StockView, StockTableSFC, PartnerStockSFC, AddOrUpdateStockSFC
- Implementation of simplified logic (first dummy data, then backend calls)
- Endpoint config in env variables

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-VITE_BASE_URL=http://localhost:8081/catena
\ No newline at end of file
+VITE_BACKEND_BASE_URL=http://localhost:8081/catena/
+VITE_ENDPOINT_MATERIALS=stockView/materials
+VITE_ENDPOINT_PRODUCTS=stockView/products
+VITE_ENDPOINT_MATERIAL_STOCKS=stockView/material-stocks
+VITE_ENDPOINT_PRODUCT_STOCKS=stockView/product-stocks
+VITE_ENDPOINT_CUSTOMER=stockView/customer?materialUuid=
+VITE_ENDPOINT_PARTNER_PRODUCT_STOCKS=stockView/partner-product-stocks
+VITE_ENDPOINT_UPDATE_PARTNER_PRODUCT_STOCK=stockView/update-partner-product-stock?materialUuid=
```
Comment (mhellmeier) [inline .env:10]: I suggest ending every file with an empty newline.

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (sengfer-ISST) [reply .env]: should be fixed with commit d7e914d251ec30214ef905cfd0e2a112471dd13b

File: src/assets/base.css
[the remark this answers is not in the document]
Comment (sengfer-ISST) [reply src/assets/base.css:1]: should be fixed with commit d7e914d251ec30214ef905cfd0e2a112471dd13b

File: package-lock.json
Comment (mhellmeier) [inline package-lock.json:1]: Due to a version change, the Eclipse dash tool should be used to check the new version and update the `DEPENDENCIES` file accordingly.

**Review by mhellmeier — CHANGES_REQUESTED** [review]
LGTM

File: package-lock.json
Comment (mhellmeier) [reply package-lock.json:1]: @sengfer-ISST : Can you please check the mentioned dependency point?

**Review by mhellmeier — APPROVED** [review]
LGTM

**Comment by mhellmeier** [comment]
Failing Quality Check pipelines will be fixed in #35.

**Pull request merged**